test/e2e/pages/configuration.page.ts

```diff
@@ -0,0 +1,81 @@
+import { Locator, Page } from '@playwright/test';
+
+type Tabs = {
+  fields: Locator;
+};
+
+export class ConfigurationPage {
+  readonly page: Page;
+  readonly projectId: string;
+
+  readonly tabs: Tabs;
+  readonly applyButton: Locator;
+
+  readonly url: string;
+
+  constructor(page: Page, projectId: string) {
+    this.page = page;
+    this.projectId = projectId;
+
+    this.tabs = {
+      fields: this.page.locator('[data-ng-repeat="tab in $ctrl.tabs"] >> text=Fields >> visible=true')
+    };
+    this.applyButton = this.page.locator('button >> text=Apply');
+
+    this.url = `/app/lexicon/${projectId}/#!/configuration`;
+  }
+
+
+  async goto() {
+    await this.page.goto(this.url);
+    // TODO: wait for an element on the page to be visible (navigation and loading pages are flaky)
+    // await this.page.waitForTimeout(3000);
+  }
+
+  async getTable(tableTitle: string): Promise<Locator> {
+    // note that at the moment, all tables are in one huge <table>
+    return this.page.locator(`table:has(th:has-text("${tableTitle}"))`);
+  }
+
+  async getRow(tableLocator: Locator, rowTitle: string): Promise<Locator> {
+    const row: Locator = tableLocator.locator(`tr:has-text("${rowTitle}"):not(:has-text("Input Systems for")) >> visible=true`);
+    // if two rows within the same table have the same name, a warning is logged to the console
+    //  note: this warning is always logged in the test "Makw taud input system.." of file editor-entry.spec.ts because of the messy table layout
```

Comment:
This is because of the structure of the table:
![image](https://user-images.githubusercontent.com/43119728/188646502-c1af5c3a-4e44-4069-adb5-4a30044ea226.png)

Reply:
Love the back-of-the-napkin investigative diagram here!!!  If you have extra time, now that you understand the messy/inconsistent table layout, you could go and change the configuration layout yourself so that the tests can use the same code.